server.py

```diff
@@ -53,6 +53,7 @@ def run_tasks(services):
     loop.run_until_complete(app_svc.register_contacts())
     loop.run_until_complete(app_svc.load_plugins(args.plugins))
     loop.run_until_complete(data_svc.load_data(loop.run_until_complete(data_svc.locate('plugins', dict(enabled=True)))))
+    loop.run_until_complete(app_svc.load_plugin_expansions(loop.run_until_complete(data_svc.locate('plugins', dict(enabled=True)))))
```

Comment:
why execute this separately? it seems it should occur when plugins are loaded

Reply:
Load order issue - I'm doing post-processing on the abilities after they have been loaded into memory.  That only happens have the `load_data` function is called, which can't be called until the load_plugins completes.

Reply:
would this be solved if we load_plugins after load_data..?

Reply:
load_data only loads data from the data_dirs of loaded plugins.

So the options as I see it are:
1. Pass a ContextVar to the plugins to allow an asyncio future call in the `enable()`
2. have a two-step approach (like I have in this PR)